### PR TITLE
Arm 0.13.0

### PR DESCRIPTION
**Pull request by sarangan12**
@vishrutshah @veronicagg @salameer Please review and approve

### CODE REVIEW

File: ChangeLog.md [hunk 1]
```diff
@@ -1,3 +1,22 @@
+## 2017.09.27 - Azure ARM version 0.13.0
+
+* Regened all azure_mgmt_* sdks using [AutoRest.2.0.4144](https://www.npmjs.com/package/autorest) for release of ARM version 0.13.0
+* Updated [azure_mgmt_cognitive_services](https://rubygems.org/gems/azure_mgmt_cognitive_services) operations Update, Delete, GetProperties, ListKeys, RegenerateKey & ListSkus from CognitiveServiceAccounts group to Accounts group. Refer [PR #989](https://github.com/Azure/azure-sdk-for-ruby/pull/989) and [PR #1707](https://github.com/Azure/azure-rest-api-specs/pull/1707) for further details.
+* Updated [azure_mgmt_compute](https://rubygems.org/gems/azure_mgmt_compute) with Vmss rolling upgrade, Vmss patch and Vm instance view APIs. Refer [PR #999](https://github.com/Azure/azure-sdk-for-ruby/pull/999) and latest [specification](https://github.com/Azure/azure-rest-api-specs/blob/current/specification/compute/resource-manager/readme.md) for further details.
```
Comment (veronicagg) [inline ChangeLog.md]: the link to the "specification" will always point to latest, if you wanted to point to the state today, you may need to get a "perma link" or a hash for the commit that you're referencing. 
I would actually recommend to add a link with the hash to the last commit you got, for the whole repo, so we know which snapshot we took. Also include the version of autorest and autorest.ruby being used.

**Comment by veronicagg** [comment]
It'd be great if you have a checklist similar to what's listed in our previous release https://github.com/Azure/azure-sdk-for-ruby/pull/962 so we know what's being done for this one.

**Comment by sarangan12** [comment]
- [x] Regen all azure_mgmt_* sdks using AzureRest.Ruby version [Autorest.Ruby.2.0.15](https://www.npmjs.com/package/@microsoft.azure/autorest.ruby)
- [x] Update ARM_VERSION file to `0.13.0`
- [x] Update swagger_to_sdk config file to version `0.13.0`
- [x] Update changelog

**Comment by sarangan12** [comment]
Manual changes:

1. https://github.com/sarangan12/azure-sdk-for-ruby/blob/58710c2f90d075d33fe7859e0dbc9535008127bb/management/azure_mgmt_datalake_analytics/lib/generated/azure_mgmt_datalake_analytics/models/sub_resource.rb

2. https://github.com/sarangan12/azure-sdk-for-ruby/blob/58710c2f90d075d33fe7859e0dbc9535008127bb/management/azure_mgmt_datalake_analytics/lib/generated/azure_mgmt_datalake_analytics.rb#L52

3. https://github.com/sarangan12/azure-sdk-for-ruby/blob/6b00d758102b33d39bb1ade254036dbd3eae8fc2/management/azure_mgmt_stream_analytics/lib/generated/azure_mgmt_stream_analytics/models/sub_resource.rb

4. https://github.com/sarangan12/azure-sdk-for-ruby/blob/6b00d758102b33d39bb1ade254036dbd3eae8fc2/management/azure_mgmt_stream_analytics/lib/generated/azure_mgmt_stream_analytics.rb#L103

5. https://github.com/sarangan12/azure-sdk-for-ruby/blob/6b00d758102b33d39bb1ade254036dbd3eae8fc2/management/azure_mgmt_stream_analytics/lib/generated/azure_mgmt_stream_analytics/models/function.rb#L12

6. https://github.com/sarangan12/azure-sdk-for-ruby/blob/6b00d758102b33d39bb1ade254036dbd3eae8fc2/management/azure_mgmt_stream_analytics/lib/generated/azure_mgmt_stream_analytics/models/input.rb#L12

7. https://github.com/sarangan12/azure-sdk-for-ruby/blob/6b00d758102b33d39bb1ade254036dbd3eae8fc2/management/azure_mgmt_stream_analytics/lib/generated/azure_mgmt_stream_analytics/models/subscription_quota.rb#L11

8.https://github.com/sarangan12/azure-sdk-for-ruby/blob/6b00d758102b33d39bb1ade254036dbd3eae8fc2/management/azure_mgmt_stream_analytics/lib/generated/azure_mgmt_stream_analytics/models/transformation.rb#L13

**Pull request closed without merging**